lib/view.js

```diff
@@ -0,0 +1,56 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var URL = require('url');
+var assert = require('assert');
+var util = require('util');
+var _ = require('lodash');
+
+module.exports = mixinView;
+
+function mixinView(Cloudant) {
+  var debug = require('debug')('loopback:connector:cloudant:view');
+
+  /**
+   * Get data returned by querying a view
+   *
+   * @param {String} ddocName The design doc name
```

Comment:
It is the third one, `{name} (db and _design/ is implied)`, and this is also what the driver function  `mo.db.viewDocs()` expects.